src/__experimental__/components/label/label.spec.js

```diff
@@ -20,6 +22,54 @@ describe('Label', () => {
     expect(render({}, TestRenderer.create)).toMatchSnapshot();
   });
 
+  describe('context handlers', () => {
+    let contextOnMouseEnter;
+    let groupContextOnMouseEnter;
+
+    let contextOnMouseLeave;
+    let groupContextOnMouseLeave;
+
+    let wrapper;
+
+    beforeEach(() => {
+      contextOnMouseEnter = jest.fn();
+      groupContextOnMouseEnter = jest.fn();
+
+      contextOnMouseLeave = jest.fn();
+      groupContextOnMouseLeave = jest.fn();
+
+      wrapper = renderWithContext({},
+        {
+          onMouseEnter: groupContextOnMouseEnter,
+          onMouseLeave: groupContextOnMouseLeave
+        },
+        {
+          onMouseEnter: contextOnMouseEnter,
+          onMouseLeave: contextOnMouseLeave
+        },);
+    });
+
+
+    it('triggers onMouseEnter callbacks passed from context', () => {
+      wrapper.find(StyledLabel).props().onMouseEnter();
+      expect(contextOnMouseEnter).toHaveBeenCalled();
+      expect(groupContextOnMouseEnter).toHaveBeenCalled();
+    });
+
+    it('triggers onMouseLeave callbacks passed from context', () => {
+      wrapper.find(StyledLabel).props().onMouseLeave();
+      expect(contextOnMouseLeave).toHaveBeenCalled();
+      expect(groupContextOnMouseLeave).toHaveBeenCalled();
+    });
+
+    it('nothing happens if onFocus, onBlur, onMouseEnter or onMouseLeave callbacks are not provided', () => {
```

Comment:
And same here. `it('does nothing if onFocus, onBlur, onMouseEnter or onMouseLeave callbacks are not provided'` would work work better.